src/core/executor.c

```diff
@@ -206,6 +207,11 @@ static int run(int argc, char *argv[]) {
         log_set_prohibit_ipc(false);
         log_open();
 
+        /* Clear ambient capabilities, so services do not inherit them implicitly. Dropping them does
+         * not affect the permitted and effective sets which are important for the executor itself to
+         * operate. */
+        capability_ambient_set_apply(0, /* also_inherit= */ false);
```

Comment:
should be cast to `(void)`, since we knowingly ignore the error condition here. as per coding style

Reply:
ah, apparently someone else already addressed this.